src/shared/api/generated/CBioPortalAPI-docs.json

```diff
@@ -2324,6 +2328,43 @@
         }
       }
     },
+    "/structuralvariant/fetch": {
```

Comment:
@inodb  this seems new and not related to my code. You know anything about this?

Reply:
it's this one i think https://github.com/cBioPortal/cbioportal/pull/4057